src/pages/iou/WaypointEditor.js

```diff
@@ -169,5 +171,22 @@ export default compose(
             key: (props) => `${ONYXKEYS.COLLECTION.TRANSACTION}${props.transactionID}`,
             selector: (transaction) => (transaction ? {transactionID: transaction.transactionID, comment: {waypoints: lodashGet(transaction, 'comment.waypoints')}} : null),
         },
+
+        recentWaypoints: {
+            key: ONYXKEYS.NVP_RECENT_WAYPOINTS,
+
+            // Only grab the most recent 5 waypoints because that's all that is shown in the UI. This also puts them into the format of data
+            // that the google autocomplete component expects for it's "predefined places" feature.
+            selector: (waypoints) =>
+                _.map(waypoints && waypoints.length ? waypoints.slice(0, 5) : [], (waypoint) => ({
```

Comment:
NAB. Just curious why we check for `waypoints.length?

Reply:
I think I was just being paranoid about calling `.slice()` on something that wasn't an array. It shouldn't ever be something other than an array, but you never know... I can remove it if you want.